valohai_yaml/objs/config.py

```diff
@@ -62,7 +72,9 @@ def get_step_by(self, **kwargs):
         return None
 
     def __repr__(self):  # pragma: no cover
-        return '<Config with %d steps: %r>' % (
+        return '<Config with %d steps and %d endpoints: %r, %r>' % (
```

Comment:
This would probably be clearer if the numbers and names were paired up, like

> Config with %d steps (%r) and %d endpoints (%r)

?

Reply:
👍